packages/renderer/package.json

```diff
@@ -6,18 +6,50 @@
   "license": "MIT",
   "private": false,
   "scripts": {
-    "build_": "yarn build:cjs && yarn build:es",
+    "build": "yarn build:cjs && yarn build:es && yarn build:dist",
     "build:cjs": "rimraf lib && tsc --module commonjs --target es2015 --outDir lib",
     "build:es": "rimraf es && tsc --module esnext --target es2015 --outDir es",
-    "clean": "rimraf lib es",
+    "build:dist": "rimraf dist && parcel build src/index.ts esnext --target es2015",
```

Comment:
esnext --target es2015 - I think this will not be applied, you are not providing the params to tsc config - only to parcel command. Just keep the correct module and target in tsconfig